src/ape_plugins/_cli.py

```diff
@@ -31,29 +42,65 @@ def cli():
     is_flag=True,
     help="Display all plugins (including Core)",
 )
+
+
+
 @ape_cli_context()
 def _list(cli_ctx, display_all):
-    plugins = []
+    installed_second = set()
+    installed_first = set()
+    installed_third = set()
     for name, plugin in plugin_manager.list_name_plugin():
-        version_str = ""
         version = get_package_version(name)
 
         if name in FIRST_CLASS_PLUGINS:
             if not display_all:
+            
                 continue  # NOTE: Skip 1st class plugins unless specified
-            version_str = " (core)"
+            installed_first.add(f"{name}")
 
-        elif version:
-            version_str = f" ({version})"
+        elif name in SECOND_CLASS_PLUGINS:
+            if not display_all:
+                continue
+            installed_second.add(f"{name}")
+
+        elif name not in FIRST_CLASS_PLUGINS or name not in SECOND_CLASS_PLUGINS:
+            installed_third.add(f"{name}    ({version})")
+        else:
+            print(name + "is not a plugin")
```

Comment:
Two options here:

1. I don't know how we would ever get here, so we could probably just remove this `else` statement.
2. If we keep the else statement, use `cli_ctx.logger.error(f"{name} is not a plugin.")`...

Reply:
I will use ``` cli_ctx.logger.error(f"{name} is not a plugin.") ```